app/assets/javascripts/unsupported-browsers.js

```diff
@@ -32,7 +32,7 @@
 
     var agent = navigator.userAgent;
     if (agent.match(/MSIE [789]\.0/) === null &&                // IE 7-9
-        agent.match(/Firefox\/(([1-2][0-9]|30)\.)/) === null) { // Firefox 10-30
+        agent.match(/Firefox\/(([1-2][0-9]|3[0-7])\.)/) === null) { // Firefox 10-37
```

Comment:
Rumors say that the version after "Windows 8" was called "Windows 10", rather than "Windows 9", because of code like this:

````
        if (osName.startsWith("Windows")) {
            isWindows = true;
            if (osName.startsWith("Windows 9") ||
                osName.startsWith("Windows Me"))
            return; // win9x/Me cannot handle long paths
        }
````
(from OpenJDK)

I can already see us failing for Firefox 100 (due in approx. August 2022)... :horse: